src/index.js

```diff
@@ -1 +1,32 @@
-export default () => true
+import defaultsDeep from 'lodash.defaultsdeep';
+import pickBy from 'lodash.pickby';
+
+class Stipulate {
+
+  constructor(options) {
+    this.baseOptions = options;
+  }
+
+  prefix(request) {
+    return request;
+  }
+
+  postfix(response) {
```

Comment:
> I agree that the name isn't particularly clear, but afterSend doesn't seem like an improvement.

I semi-agree. It's not uncommon to have `before<Action>`, `on<Action>`, and `after<Action>`, and the action here is `send`. I'm iffy with `onResponse` because are you talking about receiving the response from the server or fetch's Response? `withRequest` and `withResponse`?